recipes/multiqc/meta.yaml

```diff
@@ -7,7 +7,7 @@ source:
   sha256: 1a6bea211ab584164a6c7513812ede57b28712bc528bd34cc5f1a581e7410644
 
 build:
-  number: 1
+  number: 2
```

Comment:
Oh ... I see one problem here. It's a noarch package. This means we only produce one package and not multiple once for every python version. With a noarch package this is not possible - sorry.

Reply:
Ah, good spot. Can we just remove the `noarch` or is that needed?

Reply:
That up to the maintainer ;) We prefer noarch-packages, as we only need to host/maintain one package in contrast to 4. But if you think its worth no one is stopping you :)

Reply:
So far only one person has complained about this problem, so I'll close for now and return if more people encounter the same thing 👍

Reply:
So it looks like more people may be affected. I've reopened this PR and commented out the `noarch` line. Hopefully we can add this back at a later date (though probably only if I drop py2k support I guess).